### ISSUE TITLE
Ability to Archive whole projects

### ISSUE THREAD

**Issue by aekhendry** [thesgc/chembiohub_helpdesk#528]
I need to be able to clean up old projects that i use for tests and demos. This functionality will have to come some time for the user but the sooner the better as my project list is getting out of hand.

**Comment by strets123** [comment]
Yes needs to be addressed, but perhaps the rest of group admin not as needed as this

**Comment by KarenPorter** [comment]
@strets123 in progress - please provide update
